src/Administration/Resources/app/administration/src/app/init-post/amplitude.init.ts

```diff
@@ -50,6 +50,31 @@ export default async function (): Promise<void> {
         },
     });
 
+    const originalFetch = globalThis.fetch;
+    const analyticsGatewayUrl = Shopware.Store.get('context').app.analyticsGatewayUrl;
+    const eventUrl = `${analyticsGatewayUrl}/event`;
+
+    globalThis.fetch = async function(input: RequestInfo | URL, init?: RequestInit): Promise<Response> {
+        const url = typeof input === 'string' ? input : input instanceof URL ? input.href : input.url;
+
+        if (url.includes(eventUrl)) {
```

Comment:
Can you maybe switch this if statement so we do an early return, just so it's immediately clear that this is a work around for an edge case?